src/mame/drivers/at.cpp

```diff
@@ -5293,227 +5673,245 @@ ROM_END
 
 
 /***************************************************************************
-
   Game driver(s)
-
```

Comment:
Most heading comments like this have the blank lines between the rows of asterisks and the text.